src/lesson6_task1/MilitaryTransport.java

```diff
@@ -0,0 +1,53 @@
+package lesson6_task1;
+
+public class MilitaryTransport extends AirTransport {
+    boolean isCatapult;
+    int countRocket;
+
+    //конструктор
+    public MilitaryTransport(int powerHours, String brand, int maxSpeed, int minRunwayLength,
+                             float weight, float wingspan, int countRocket, boolean isCatapult) {
+        super(powerHours, maxSpeed, brand, weight, wingspan, minRunwayLength);
+        this.countRocket = countRocket;
+        this.isCatapult = isCatapult;
+    }
+
+    @Override
+    public String getDescription() {
+        this.technicalChar = "Марка " + this.brand + ":\n-Мощность " + this.powerHours + " лошадиных сил;\n-Мощность " + getPowerKw() + " кВ;\n" +
+                "-Максимальная скорость " + this.maxSpeed + " км/ч;\n-Масса " + this.weight + " кг;\n" +
+                "-Размах крыльев " + this.wingspan + " м;\n-Минимальная длина взлётно-посадочной полосы для взлёта " + this.minRunwayLength + " м;" +
+                "\n-" + checkCatapultSystem() + "\n-Количество ракет на борту - " + this.countRocket + " штук.\n";
+        return this.technicalChar;
```

Comment:
@Nadya101 Сокращаем код
```suggestion
        return "Марка " + this.brand + ":\n-Мощность " + this.powerHours + " лошадиных сил;\n-Мощность " + getPowerKw() + " кВ;\n" +
                "-Максимальная скорость " + this.maxSpeed + " км/ч;\n-Масса " + this.weight + " кг;\n" +
                "-Размах крыльев " + this.wingspan + " м;\n-Минимальная длина взлётно-посадочной полосы для взлёта " + this.minRunwayLength + " м;" +
                "\n-" + checkCatapultSystem() + "\n-Количество ракет на борту - " + this.countRocket + " штук.\n";
```

Reply:
Исправлено)